.github/workflows/ci-release.yml

```diff
@@ -9,7 +9,7 @@ on:
 # and https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions
 permissions:
   deployments: write
-  contents: read
+  contents: read,write
```

Comment:
oh-oh
```
The workflow is not valid. .github/workflows/ci-release.yml (Line: 12, Col: 13): Unexpected value 'read,write'
```

Reply:
maybe this needs an array syntax `["read","write"]`?

Reply:
yeah,  would expect write to include read

Reply:
but also curious what "content" this is writing (GH documentation is so bad for these, they just list them without explanation)

Reply:
probably this: `PATCH /repos/{owner}/{repo}/releases/assets/{asset_id} (write)`